frontend/dispatcher.lua

```diff
@@ -154,6 +154,9 @@ local settingsList = {
     ----
 
     -- Reader
+    show_alarm_menu = {category="none", event="ShowAlarmMenu", title=_("Set reader alarm"), reader=true},
+    show_timer_menu = {category="none", event="ShowTimerMenu", title=_("Set reader timer"), reader=true},
+    stop_timer = {category="none", event="StopTimer", title=_("Stop reader timer"), reader=true},
```

Comment:
Plugins do this in the following way, for example:
https://github.com/koreader/koreader/blob/3e39740a42983f390b6c32f0f92a410c452569bf/plugins/statistics.koplugin/main.lua#L112-L113

Reply:
General like dictionary.